plugins/callback/influxdb.py

```diff
@@ -0,0 +1,160 @@
+# (c) 2019, Jason Neurohr, <jason@jasonneurohr.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: influxdb
+    callback_type: aggregate
+    requirements:
+      - whitelist in configuration
+    short_description: Adds play duration to InfluxDB
+    version_added: "1.3.0"
+    description:
+        - This callback captures the total play duration and submits it to InfluxDB.
+    options:
+      influxdb_addr:
+        description: "InfluxDB address in the form FQDN, FQDN:PORT, IP, or IP:PORT."
+        env:
+          - name: INFLUXDB_ADDR
+        ini:
+          - section: callback_influxdb
+            key: influxdb_addr
+      influxdb_db:
+        description: InfluxDB database name.
+        env:
+          - name: INFLUXDB_DB
+        ini:
+          - section: callback_influxdb
+            key: influxdb_db
+      influxdb_username:
+        description: InfluxDB database user name.
+        env:
+          - name: INFLUXDB_USERNAME
+        ini:
+          - section: callback_influxdb
+            key: influxdb_username
+      influxdb_password:
+        description: InfluxDB database password.
+        env:
+          - name: INFLUXDB_PASSWORD
+        ini:
+          - section: callback_influxdb
+            key: influxdb_password
+      influxdb_tls:
+        description: Use TLS to connect to InfluxDB (HTTPS).
+        env:
+          - name: INFLUXDB_TLS
+        ini:
+          - section: callback_influxdb
+            key: influxdb_tls
+        default: True
+        type: bool
+      influxdb_validate_cert:
+        description: Validate the TLS certificate of the InfluxDB server (for HTTPS URLs).
+        env:
+          - name: INFLUXDB_VALIDATE_CERT
+        ini:
+          - section: callback_influxdb
+            key: influxdb_validate_cert
+        default: True
+        type: bool
+'''
+
+EXAMPLES = '''
+examples: >
```

Comment:
Since you care about line-breaks:
```suggestion
examples: |
```